proposal/forward-with.md

```diff
@@ -0,0 +1,170 @@
+# Reconfigurable Modules via "@forward ... with": Draft 1
+
+*([Issues](https://github.com/sass/sass/issues/2744))*
+
+## Table of Contents
+
+- [Reconfigurable Modules via "@forward ... with": Draft 1](#reconfigurable-modules-via-%22forward--with%22-draft-1)
+  - [Table of Contents](#table-of-contents)
```

Comment:
Nit: We usually don't include the title or the table of contents in the table of contents. See other proposals for examples.

Reply:
heh, oops: I think something in my code editor did that automatically and I didn't catch it.